pyproject.toml

```diff
@@ -97,15 +97,16 @@ dependencies = [
   "pytest-lazy-fixture",
   "arize",
   "langchain>=0.0.257",
-  "llama-index",
+  "llama-index>=0.8.1",
   "openai",
   "tenacity",
+  "nltk==3.8.1", 
```

Comment:
this gets imported by llama_index and I can't make it not fail the test. Gonna punt for now.